core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -212,14 +212,14 @@ Rename `variant` to `mode-variant` and change type name.
 
 ```jsx
 // Old ❌
-<sdds-button variant="variant">
+<tds-button variant="variant">
```

Comment:
I think this should still be sdds, since that's the "old" way.

Reply:
Good catch!